hooks: Add `exclude_datas` to utils.hooks.collect_all(). (#5201)
pyinstaller/pyinstaller: news/5201.hooks.rst

> Hook-gevent passes filter_submodules to ``utils.hooks.collect_all()`` to filter out those submodules.

Reviewer: News items should just say *what* has changed rather than *how*. 
```suggestion
Modify hook for ``gevent`` to exclude test submodules.
```
